buildSrc/src/main/java/co/icanteach/projectx/buildsrc/Classpaths.kt

```diff
@@ -1,5 +1,9 @@
 object Classpaths {
     val gradleClasspath = "com.android.tools.build:gradle:${Versions.gradleVersion}"
-    val kotlinGradleClasspath = "org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.kotlinVersion}"
-    val gradleVersionPlugin = "com.github.ben-manes:gradle-versions-plugin:${Versions.gradleVersionPluginVersion}"
+    val kotlinGradleClasspath =
+        "org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.kotlinVersion}"
```

Comment:
The formatting  of `kotlinGradleClasspath` and `gradleVersionPlugin` is different from previous commits. These may be fit 1 line as other dependencies.